helm/oncall/templates/integrations/deployment.yaml

```diff
@@ -0,0 +1,107 @@
+{{- if .Values.detached_integrations.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "oncall.detached_integrations.fullname" . }}
+  labels:
+    {{- include "oncall.detached_integrations.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.detached_integrations.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "oncall.detached_integrations.selectorLabels" . | nindent 6 }}
+  strategy:
+    {{- toYaml .Values.detached_integrations.updateStrategy | nindent 4 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        random-annotation: {{ randAlphaNum 10 | lower }}
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "oncall.detached_integrations.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "oncall.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      initContainers:
+        {{- include "oncall.initContainer" . | indent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{- if .Values.oncall.devMode }}
+          command: ["sh", "-c", "uwsgi --disable-logging --py-autoreload 3 --ini uwsgi.ini"]
+          {{- end }}
+          ports:
+            - name: http
+              containerPort: 8080
+              protocol: TCP
+          env:
+            {{- include "snippet.oncall.env" . | nindent 12 }}
+            {{- include "snippet.oncall.slack.env" . | nindent 12 }}
+            {{- include "snippet.oncall.telegram.env" . | nindent 12 }}
+            {{- include "snippet.oncall.smtp.env" . | nindent 12 }}
+            {{- include "snippet.oncall.twilio.env" . | nindent 12 }}
+            {{- include "snippet.oncall.exporter.env" . | nindent 12 }}
+            {{- include "snippet.db.env" . | nindent 12 }}
+            {{- include "snippet.broker.env" . | nindent 12 }}
+            {{- include "oncall.extraEnvs" . | nindent 12 }}
```

Comment:
_nit_ for this portion I would suggest creating a new helper snippet in `helm/oncall/_env.tpl` which does all of these `include` statements for us (we have this same block duplicated [here](https://github.com/grafana/oncall/blob/dev/helm/oncall/templates/celery/deployment.yaml#L62-L70) and [here](https://github.com/grafana/oncall/blob/dev/helm/oncall/templates/engine/deployment.yaml#L47-L55))